stories/cosmoz-autocomplete.stories.js

```diff
@@ -104,3 +104,29 @@ export const wrap = () => html` ${css}
 		wrap
 		style="max-width: 170px"
 	></cosmoz-autocomplete>`;
+
+export const contour = () => html`<style>
+		@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;300;400;500&display=swap');
+		cosmoz-autocomplete,
+		cosmoz-listbox {
+			font-family: 'Inter', sans-serif;
+		}
```

Comment:
Why did you include a font here?

Reply:
It is a style spec from the FIgma mockups and UX meetings, and I noticed some issues in fixing this font from frontend.

Reply:
But there is already the `css` that the other stories use. Why don't you use that one?

Reply:
There seems to be something wrong with the `@import` rule there. Anyways I moved the font definition there, the font import I need is performed in Frontend.

Reply:
You are duplicating the same code that is in the `css` variable at the top of this file.
https://github.com/Neovici/cosmoz-autocomplete/blob/ab57478cef04c6df23834b148ec7a7252461ce1d/stories/cosmoz-autocomplete.stories.js#L9-L16

Why aren't you just interpolating that value, just like in the other stories? Why do you need to duplicate it?

Pay attention to the fact that it is already wrapped in a `<style>` tag. You might have tried to do something like `<style>${css}</style>`, in which case it would have resulted in `<style><style>...</style></style>`, which of course won't work properly.

Reply:
OK, consistently with the other comment in the style file, it's probably best if this font styling is set directly in frontend, via e.g. vars - I recall you mentioning the latter being more effective, I will add a var for it so it's accessible from the outside.

Reply:
Or, to avoid adding more code, already existing parts might be a better solution. I recall you mentioning that vars are more effective than parts, but they're already used and working in frontend.